ui/src/app/machines/views/MachineDetails/MachineCommissioning/MachineCommissioningTable/MachineCommissioningTable.test.tsx

```diff
@@ -0,0 +1,162 @@
+import { mount } from "enzyme";
+import { Provider } from "react-redux";
+import { MemoryRouter } from "react-router";
+import configureStore from "redux-mock-store";
+
+import MachineCommissioningTable from ".";
+
+import type { RootState } from "app/store/root/types";
+import {
+  machineState as machineStateFactory,
+  machineDetails as machineDetailsFactory,
+  partialScriptResult as partialScriptResultFactory,
+  rootState as rootStateFactory,
+  scriptResult as scriptResultFactory,
+  scriptResultState as scriptResultStateFactory,
+} from "testing/factories";
+
+const mockStore = configureStore();
+
+describe("MachineCommissioningTable", () => {
+  let state: RootState;
+  beforeEach(() => {
+    state = rootStateFactory({
+      machine: machineStateFactory({
+        loaded: true,
+        items: [
+          machineDetailsFactory({
+            locked: false,
+            permissions: ["edit"],
+            system_id: "abc123",
+          }),
+        ],
+      }),
+      scriptresult: scriptResultStateFactory({
+        loaded: true,
+      }),
+    });
+  });
+
+  it("renders", () => {
```

Comment:
This creates an extremely brittle 700 line snapshot. I'd be inclined to remove it.

Reply:
Like it, done